ofrak_core/ofrak/ofrak_cli.py

```diff
@@ -275,11 +285,102 @@ def handler(ofrak_env: OFRAKEnvironment, args: Namespace):
             print(dependency_info)
 
 
+class GUISubCommand(OFRAKSubCommand):
+    def create_parser(self, ofrak_subparsers):
+        gui_parser = ofrak_subparsers.add_parser(
+            "gui",
+            help="Launch the OFRAK GUI server.",
+            description="Launch the OFRAK GUI server.",
+            formatter_class=ArgumentDefaultsHelpFormatter,
+        )
+        gui_parser.add_argument(
+            "-H",
+            "--hostname",
+            action="store",
+            help="Set GUI server host address.",
+            default="127.0.0.1",
+        )
+        gui_parser.add_argument(
+            "-p",
+            "--port",
+            action="store",
+            type=int,
+            help="Set GUI server host port.",
+            default=8080,
+        )
+        gui_parser.add_argument(
+            "-b",
+            "--backend",
+            action="store",
+            help="Set GUI server backend.",
+            default=None,
+        )
+        gui_parser.add_argument(
+            "-v",
+            "--verbose",
+            action="store_true",
+            help="Enable verbose mode for debugging",
+            default=None,
+        )
+        gui_parser.add_argument(
+            "-q",
+            "--quiet",
+            action="store_true",
+            help="Enable quiet mode to minimize logging",
+            default=None,
+        )
+        return gui_parser
+
+    @staticmethod  # pragma: no cover
+    def handler(ofrak_env: OFRAKEnvironment, args: Namespace):
+        if args.verbose is True:
+            ofrak = server.OFRAK(server.logging.DEBUG)
+
+        elif args.quiet is True:
+            ofrak = server.OFRAK(server.logging.WARNING)
+
+        else:
+            ofrak = server.OFRAK(server.logging.INFO)
+
+        if args.backend is not None:
```

Comment:
This whole block of code is duplicated from `server.py`, we could probably easily refactor to not repeat this. Perhaps the body of `create_parser` should actually be defined in `server.py` (so we can use the same argparse in the `__main__` there) and then we just call it here; or can we get rid of that `__main__` block in `server.py` entirely? Seeing as we have a CLI to open the GUI here...

Reply:
That makes sense. I'm in favor of removing the `__main__` block from `server.py` since the path to start the server should be to use the CLI. This way we can remove a redundant method for starting the server that doesn't provide any additional functionality.

Reply:
That makes sense, I'm in favor of that as well.